src/Microsoft.Diagnostics.Monitoring.Options/ExceptionsConfiguration.cs

```diff
@@ -13,25 +13,28 @@ public sealed class ExceptionsConfiguration
         /// Each configuration is a logical OR, so if any of the configurations match, the exception is shown.
         /// </summary>
         [JsonPropertyName("include")]
-        public List<ExceptionFilter> Include { get; set; } = new();
+        public List<ExceptionFilter>? Include { get; set; } = new();
 
         /// <summary>
         /// The list of exception configurations that determine which exceptions should be shown.
         /// Each configuration is a logical OR, so if any of the configurations match, the exception isn't shown.
         /// </summary>
         [JsonPropertyName("exclude")]
-        public List<ExceptionFilter> Exclude { get; set; } = new();
+        public List<ExceptionFilter>? Exclude { get; set; } = new();
     }
 
-    public sealed class ExceptionFilter : IMethodDescription
```

Comment:
NOTE: We no longer derive from `IMethodDescription` as it is not a good match. All fields in `IMethodDescription` are required. All fields in `ExceptionFilter` are optional.